### PR TITLE
Redesign Privacy Policy and Terms of Use

### PR DESCRIPTION
**Pull request by scottbommarito**
Added the Privacy Policy and Terms of Use to the redesign. Was fairly simple--just text formatting.

We will potentially need to revisit the actual text of the Privacy Policy and Terms of Use at some point. All I did was change "NuGet Gallery" to "NuGet" and "Web site(s)" to "Website(s)" as in the spec.

### CODE REVIEW

**Comment by skofman1** [comment]
Please add a list of the accessibility tests you did, and add Sam to the review.

**Comment by scottbommarito** [comment]
@skofman1 what is Sam's GitHub?

**Comment by scottbommarito** [comment]
@skofman1 Accessibility tests are pending @jonwchu's discussion with an accessibility guru regarding the expected behavior of screen reader with large bodies of text.

**Comment by microsoftsam** [comment]
How do you test these pages? Clicking on Terms of Use / Privacy down the bottom of the page leads me to the old Nuget terms page. 
I've checked out your feature branch.

**Comment by scottbommarito** [comment]
Clicking Terms of Use / Privacy should be all that's necessary. Are you sure you're on `sb-pages` and not `feature-redesign`?

I just pushed the autogenerated css files, which may have been causing trouble as well. Can you try again?

**Comment by microsoftsam** [comment]
Okay sweet, I'm seeing those changes now

**Comment by microsoftsam** [comment]
Still trying to get access to the wireframes for the site, but there really should be a bigger padding between the content and the footer of these pages:

screen shot 2017-05-31 at 4 18 28 pm

**Comment by microsoftsam** [comment]
Looks good. Add a mailto link to info@nuget.org in the contacting us section of privacy.md

File: src/Bootstrap/dist/css/bootstrap-theme.css [hunk 1]
```diff
@@ -207,4 +207,21 @@ body {
 .page-sign-in .or-row {
   margin: 30px;
 }
+.page-pages {
+  padding-bottom: 45px;
```
Comment (microsoftsam) [inline src/Bootstrap/dist/css/bootstrap-theme.css]: You might find this to be a common trend across pages. If you find this to be the case, add the padding-bottom on the container class

File: src/NuGetGallery/NuGetGallery.csproj [hunk 2]
```diff
@@ -33,6 +33,7 @@
     <MvcBuildViews Condition=" '$(MvcBuildViews)' == '' ">false</MvcBuildViews>
     <NuGetPackageImportStamp>
     </NuGetPackageImportStamp>
+    <Use64BitIISExpress />
```
Comment (loic-sharma) [inline src/NuGetGallery/NuGetGallery.csproj]: Is this intentional?

Comment (scottbommarito) [inline src/NuGetGallery/NuGetGallery.csproj]: It was automatically added, not sure if it impacts anything.

**Comment by loic-sharma** [comment]
If you end up adding a mailto like Sam suggested, you should also change the second `info@nuget.org` email in the first paragraph of the privacy statement.

**Comment by scottbommarito** [comment]
Added mailto links!

**Pull request merged**